docs/ops/state/savepoints.md

```diff
@@ -94,7 +94,7 @@ When triggering a savepoint, a new savepoint directory is created where the data
 <strong>Attention:</strong> The target directory has to be a location accessible by both the JobManager(s) and TaskManager(s) e.g. a location on a distributed file-system.
```

Comment:
Can we change this to "...location on a distributed file-system or Object Store." ?